src/api/createDeviceMessageHandlingSystem/src/v1/steps.py

```diff
@@ -191,10 +199,10 @@ def set_http_status(data, cache) -> tuple[HTTPStatus, dict]:
     check_for_existing_mhs,
     read_spine_mhs_questionnaire,
     read_device_reference_data,
-    validate_spine_mhs_questionnaire_response,
     create_mhs_device,
     create_party_key_tag,
     create_cpa_id_keys,
+    validate_spine_mhs_questionnaire_response,
```

Comment:
it was only that this stage needed access to the device ods code, and I thought to keep the code internally consistent better to take the ods code from the device rather than parse it indirectly